main.go

```diff
@@ -290,7 +292,7 @@ func startHTTPServer() *http.Server {
 	http.Handle("/metrics", promhttp.Handler())
 
 	go func() {
-		if err := srv.ListenAndServe(); err != http.ErrServerClosed {
+		if err := srv.ListenAndServe(); errors.Is(err, http.ErrServerClosed) {
```

Comment:
This is inverting the conditional.

```suggestion
		if err := srv.ListenAndServe(); !errors.Is(err, http.ErrServerClosed) {
```

Reply:
Oh right - thanks for noticing that.